service/mantle/shipstation/ShipStationServices.xml

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<services xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:noNamespaceSchemaLocation="http://moqui.org/xsd/service-definition-2.1.xsd">
+
+    <!--
+    TODO:
+        Time conversion for default time zone.
+    -->
+
+    <!--Service for getting XML data from screen-->
+    <service verb="get" noun="OrderData">
+        <in-parameters>
+            <parameter name="start_date" type="Timestamp" format="MM/dd/yyyy HH:mm">
+                <description>Find orders placed at or after this date.</description>
+            </parameter>
+            <parameter name="end_date" type="Timestamp" format="MM/dd/yyyy HH:mm">
+                <description>Find orders placed at or before this date.</description>
+            </parameter>
+            <parameter name="page"/>
+        </in-parameters>
+        <actions>
+
+            <set field="Orders" from="[]"/>
+
+            <!--Rendering FTL and writting Data from FTL to response-->
+            <script>
+                def response = ec.web.getResponse();
+                def context =
+                ec.service.sync().name("co.hotwax.oms.OrderServices.find#Orders").parameters([statusId:'OrderApproved',
+                placedDateStart:ec.l10n.format(start_date,'yyyy-MM-dd
+                HH:mm'),placedDateEnd:ec.l10n.format(end_date,'yyyy-MM-dd HH:mm')]).call()
+
+                Orders = context.orders
+                response.setContentType("application/xml")
+                Writer w = response.getWriter()
+                ec.resource.template("component://mantle-shipstation/screen/OrderData.xml.ftl",w)
+                w.flush()
+                w.close()
+            </script>
+
+        </actions>
+    </service>
+
+    <!-- service for POST call, when ShipStation wants to POST Shipment Notification-->
+    <service verb="post" noun="ShipNotification">
```

Comment:
Deepak sir's Comments:
1. The service to create Shipment for Order when ShipStation Notify should create Shipment using OOTB service where we can give Item details so that only these items will be marked completed because right now some of the cases might be missed to be handled when using these services.
2. Such service doesn't exist in Moqui ShipmentServices, therefore, make a new service for this task.

Reply:
Sounds good. Thanks for providing the details @siddharth0801 

I totally agree with the suggestion given by @dixitdeepak

Reply:
Thank you, sir, I'm working on this custom service.